cookbooks/nginx/templates/default/nginx_app.conf.erb

```diff
@@ -82,7 +82,10 @@ server {
   #
   # Log files are stored within the /var/log/engineyard/nginx/ directory.
   #
-  <% if @ssl %>
+  <% if @syslog_enabled %>
+  access_log <%= @syslog_path %>;
+  error_log <%= @syslog_path %>;
+  <% elsif @ssl %>
```

Comment:
@wildbillcat Why this? Is there a reason we don't want to log to syslog when ssl is enabled?